src/CoreFx.Private.TestUtilities/src/System/PlatformApis.Unix.cs

```diff
@@ -0,0 +1,260 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace System
+{
+    internal static class PlatformApis
+    {
```

Comment:
All this is comming from PlatformAbstractions?

Reply:
Yeah that's a direct copy. I just removed the Windows relevant pieces from it.